pkg/appgw/requestroutingrules_test.go

```diff
@@ -0,0 +1,115 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package appgw
+
+import (
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var expectedRedirectID = "/subscriptions/" + testFixturesSubscription +
+	"/resourceGroups/" + testFixtureResourceGroup +
+	"/providers/Microsoft.Network/applicationGateways/" + testFixtureAppGwName +
+	"/redirectConfigurations/" + agPrefix + "sslr-" +
+	testFixturesNamespace +
+	"-" +
+	testFixturesName
+
+func makeHTTPURLPathMap() network.ApplicationGatewayURLPathMap {
+	return network.ApplicationGatewayURLPathMap{
+		Name: to.StringPtr("-path-map-name-"),
+		ApplicationGatewayURLPathMapPropertiesFormat: &network.ApplicationGatewayURLPathMapPropertiesFormat{
```

Comment:
url path map also has mandatory default redirect or default backend pool/backendhttpsetting

Reply:
I'm so glad you pointed this out.  I commented this out AND moved the creation of these structs in test_fixtures